estafette/estafette-cloudflare-dns
# added helm chart

Steps to test

- install https://github.com/kubernetes/helm/ on your machine
- install the helm server in your kubernetes cluster with `helm init --history-max 25 --upgrade`
- lint helm chart with `helm lint chart/estafette-cloudflare-dns`
- package helm chart `helm package chart/estafette-cloudflare-dns --version 1.0.103`
- dry run helm chart install/upgrade with `helm upgrade estafette-cloudflare-dns estafette-cloudflare-dns-1.0.103.tgz --namespace estafette --install --dry-run --debug --set cloudflareApiEmail=DEF --set cloudflareApiKey=b@server.com --set rbac.create=true`

The output should look pretty much like what's in `kubernetes.yaml` and `rbac.yaml` combined.
